packages/fhir-converter/src/templates/cda/Sections/Payers.hbs

```diff
@@ -40,7 +40,7 @@
                     Example: https://github.com/HL7/C-CDA-Examples/blob/fb1c33b3a880fc47407f25c5dafa0066996529f4/Documents/CCD/CCD%201/CCD.XML
                  --}}
                     {{#if (contains (toJsonString performerEntry.templateId) "2.16.840.1.113883.10.20.22.4.87")}}
-                        {{#if performerEntry.assignedEntity.representedOrganization.name._}}
+                        {{#if performerEntry.assignedEntity.representedOrganization}}
```

Comment:
why was this changed? won't we just end up with a bunch of useless/nameless org objects now?

Reply:
The problem is Coverage needs a payor (OrgRef). So i created the nameless Org so it has something to reference. What I could do instead is not create it and if the Org doesnt exist make the payor like this

`        
"payor": [
    {
    "reference": "UNK"
    }
],`